<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>frmAPI</class>
 <widget class="QWidget" name="frmAPI">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>508</width>
    <height>413</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>frmAPI</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="1">
    <widget class="QPushButton" name="btnAppLog">
     <property name="text">
      <string>日志输出</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QPushButton" name="btnNTPAPI">
     <property name="text">
      <string>Ntp校时</string>
     </property>
    </widget>
   </item>
   <item row="0" column="2">
    <widget class="QPushButton" name="btnSaveRunTime">
     <property name="text">
      <string>启动存储运行时间</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="0" column="3">
    <widget class="QPushButton" name="btnScreenAPI">
     <property name="text">
      <string>截图工具</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QPushButton" name="btnLEDAPI">
     <property name="text">
      <string>LED取模</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QPushButton" name="btnFrameless">
     <property name="text">
      <string>无边框窗体</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
